.github/workflows/build-wheels-linux.yml

```diff
@@ -42,6 +42,7 @@ jobs:
             post-script: build/packaging/post_build_script.sh
             smoke-test-script: build/packaging/smoke_test.py
             package-name: executorch
+            python-version: '3.10'
```

Comment:
Pinning pip version is fine but I don't think `python-version` is a legit input parameter here.  The workflow is failing https://github.com/pytorch/executorch/actions/runs/9022911297

The list of supported inputs can be found here https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_wheels_linux.yml#L6-L77

Reply:
I have a task in my queue to add support for this though, so that different repos could specify which python versions they want instead of strictly following PyTorch build matrix, which covers all 3.8 to 3.12 atm